config/database.yml

```diff
@@ -80,6 +80,7 @@ test:
 #
 production:
   <<: *default
-  database: convene_web_production
-  username: convene_web
-  password: <%= ENV['CONVENE_WEB_DATABASE_PASSWORD'] %>
+  url: <%= ENV['DATABASE_URL'] %>
+  # database: convene_web_production
+  # username: convene_web
+  # password: <%= ENV['CONVENE_WEB_DATABASE_PASSWORD'] %>
```

Comment:
Let's just delete these insead of commenting them out.

Reply:
Done!